docs/palettes/timing-with-palettes.md

```diff
@@ -85,4 +88,6 @@ palette recall.
 \<Macro\>, \[Record\], press a button to store your macro on. Then press
 \<Palette\>, \[Master Time\], \<3\> (for 3 sec), \<Exit\>, \<Macro\>.
 Repeat these steps to create a Palette Snap button (0 sec), or a 1 sec
-fade button, and so on.
+fade button, and so on. \
+
+The **Show Library** provides some pre-programmed fade times for you.
```

Comment:
We could link to the [Show Library](https://manual.avolites.com/docs/next/titan-basics/show-library) at this point
```suggestion
The **[Show Library](../titan-basics/show-library.md)** provides some pre-programmed fade times for you.
```